stock_picking_report_valued/models/stock_move_line.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2014-2018 Tecnativa - Pedro M. Baeza
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2018 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# Copyright 2016-2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = 'stock.move.line'
+
+    sale_line = fields.Many2one(
+        related='move_id.sale_line_id', readonly=True,
+        string='Related order line',
+        related_sudo=True,  # See explanation for sudo in compute method
```

Comment:
related_sudo is True by default. Please remove all entries

Reply:
@rousseldenis I did it